ProcessMaker/processmaker
# Fix Unauthorized Access Error When Editing Screen Template

This PR resolves an issue where users with all permissions encounter an `"This action is unauthorized"` error when attempting to edit a screen template under the "My Templates" tab. The problem stemmed from incorrect middleware usage on the route responsible for retrieving the screen template ID.

## Solution

- Implemented the correct middleware on the route handling the retrieval of screen template IDs.

## How to Test

1. Create a user and assign all permissions to the user.
2. Log in as the user.
3. Create a screen template.
4. Navigate to the "My Templates" tab.
5. Select "Edit Template" from the Option menu.
6. Verify that you are redirected to the screen builder without encountering any errors.

ci:next
ci:deploy

## Related Tickets & Packages
- Link to any related FOUR tickets, PRDs, or packages

## Code Review Checklist
